client/src/inside/components/page/ehr-helper.js

```diff
@@ -86,14 +86,19 @@ export default class EhrHelpV2 {
     }
     if (!data) {
       console.error('ERROR call to getActiveData when there is none')
+      throw new Error('ERROR call to getActiveData when there is none')
```

Comment:
Now that this error message may appear to the user we need to make it useful. Change text to something like
'The page with key ${this.pagekey} has no form or tables. Tell your EdEHR admin about this problem.'